birdhouse/components/README.rst

```diff
@@ -8,6 +8,76 @@ Birdhouse Components
 Scheduler
 =========
 
+The scheduler component runs specific jobs on a schedule. This is similar to using the `cron`
+service but this runs in docker containers and is specifically designed to interact with the
+Birdhouse stack.
+
+Scheduler jobs can be enabled by sourcing specific ``*.env`` files in the ``components/scheduler``
+directory. The jobs that come included with the Birdhouse code are as follows:
+
+* Enable automatic deployment:
```

Comment:
Spacing for all of these does not seem to render correctly.
Probably needs a newline between the nested lists/code blocks.

I could see an anchor to specific subsections of the doc being useful here, so maybe consider using `^^^` header/separators here as well.